### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by Elaborendum**
- Crowdin: Logo and loading bar when loading
- GMaps:
  - My Maps: Icons in main screen (checkmarks, favourites star, among others), sidebar, zoom controls and logo on the map.
  - Main: Your Data icon and main logo on the sidebar, plus the close button on the "choose your language" panel
- Matrix.to: Small logo at the bottom
- GScholar: Sidebar logo and main logo in a search

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -6503,6 +6505,23 @@ a[href*="about/products"]
 img[src$="menu_black_24dp.png"]
 #assistive-chips .e2moi img
 button[guidedhelpid="searchbutton"]
+img[src="https://mt.googleapis.com/vt/icon/name=icons/onion/1491-wht-line-blank.png&filter=ff000000"]
+div[role="checkbox"] > div
+div[aria-label="Toggle star"][role="checkbox"]
+div.uVccjd.HzV7m-pbTTYe-KoToPc-ornU0b-hFsbo.HzV7m-KoToPc-hFsbo-ornU0b
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6511]: Their are quite a lot of "nonsense" css selectors here, given these are google maps, they tend to change quite a lot.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -6503,6 +6505,23 @@ a[href*="about/products"]
 img[src$="menu_black_24dp.png"]
 #assistive-chips .e2moi img
 button[guidedhelpid="searchbutton"]
+img[src="https://mt.googleapis.com/vt/icon/name=icons/onion/1491-wht-line-blank.png&filter=ff000000"]
+div[role="checkbox"] > div
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6509]: This is quite a broad selector.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Elaborendum) [reply src/config/dynamic-theme-fixes.config]: Are you referring to that last line in particular or overall?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: > Are you referring to that last line in particular or overall?

Overall.

**Comment by Elaborendum** [comment]
I'll submit a pull request specifically for GMaps later, but I think the other ones here are fine already.

**Review by Gusted — APPROVED** [review]

**Pull request merged**